pypeit/core/extract.py

```diff
@@ -1075,8 +1083,10 @@ def create_skymask_fwhm(sobjs, thismask):
 def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, maxdev=2.0, ir_redux=False, spec_min_max=None,
             hand_extract_dict=None, std_trace=None, extrap_npoly=3, ncoeff=5, nperslit=None, bg_smth=5.0,
             extract_maskwidth=4.0, sig_thresh=10.0, peak_thresh=0.0, abs_thresh=0.0, trim_edg=(5,5),
+            boxcar_rad_pix=None,
```

Comment:
I think we should just set a default value here, which for the moment can be just the same as fwhm. I also suggest changing the name to boxcar_rad_skymask  or something like that since we are propagating this into the sobjs. That is also a question, do we need to propagate this into the sobjs?  It seems we have then multiple boxcar radii in that structure then.

Reply:
taken out of `specobjs` and renamed.